### ISSUE TITLE
can't work with home-assistant because of the acc is 0

### ISSUE THREAD

**Issue by Ming91** [owntracks/android#546]
I use owntracks for presence detection with home assistant and MQTT. And it works fine with my iPhone and two ibeacons in two deferent rooms. (Although it sometimes has beacon signal overlapping problems. blame for the small apartment xD.)

But on my android device it has a big issue of 'leave' detection. It did send message to the event topic like this:
```
{
  "_type" : "transition",
  "tid" : "mi",
  "acc" : 0.0,
  "desc" : "MR",
  "event" : "leave",
  "lat" : xxxx,
  "lon" : xxxx,
  "tst" : xxxx,
  "wtst" : xxxx,
  "t" : "b"
}
```
But on home assistant log, it shows:
`Ignoring transition update because GPS accuracy is zero: {'_type': 'transition', 'tid': 'mi', 'acc': 0.0, 'desc': 'MR', 'event': 'leave', 'lat': xxxx, 'lon': xxxx, 'tst': xxxx, 'wtst': xxxx, 't': 'b', 'topic': 'owntracks/ming/Redmi/event'}`

The 'enter' event looks all the same except the event changed to 'enter', but the hass log never shows the warning message about 'enter' event. What should i do to let hass receive the 'leave' event? In the log of ios MQTT message ,the acc field is not 0.0. so maybe there is a way to change the acc in payload?

And there is another error message in hass:
```
Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/home/ming/Code/homeassistant/lib/python3.6/site-packages/homeassistant/components/device_tracker/owntracks.py", line 93, in async_handle_mqtt_message
    yield from async_handle_message(hass, context, message)
  File "/home/ming/Code/homeassistant/lib/python3.6/site-packages/homeassistant/components/device_tracker/owntracks.py", line 504, in async_handle_message
    yield from handler(hass, context, message)
  File "/home/ming/Code/homeassistant/lib/python3.6/site-packages/homeassistant/components/device_tracker/owntracks.py", line 459, in async_handle_waypoints_message
    yield from async_handle_waypoint(hass, name_base, wayp)
  File "/home/ming/Code/homeassistant/lib/python3.6/site-packages/homeassistant/components/device_tracker/owntracks.py", line 420, in async_handle_waypoint
    rad = waypoint['rad']
KeyError: 'rad'
```

* **App build number**: 
1.2.7(12701)
* **Android version**:    
7.1.2
* **Device**: 
Redmi 5A

**Comment by jpmens** [comment]
I don't know why I asked. I should have paid more attention: your software is saying

```
KeyError: 'rad'
```

there's something missing or it's expecting something we don't give it. Type = _transition doesn't have an element `rad`, so that appears to be a HomeAssistant issue. Please ask them.

**Comment by Ming91** [comment]
@jpmens Yeah, I noticed that. I think it's related to my Hass zone settings, and it's not owntracks problem. But I doubt it is relevant to the 'leave' detection problem. Btw, my Android device can work partially with two beacons. Since it only detects the 'enter' event, it can change the status on HASS to the room with the beacon. But it can never change the status to 'Away'. 
Thank you for your patience. It's really a great app. And I tested your 2.0 beta on google play, but it seems it can't detect my beacons ,so i turn back to 1.2.7.
